src/services/ContributingDeveloperAuditService/ContributingDeveloperAuditService.ts

```diff
@@ -0,0 +1,84 @@
+import SOOSHooksApiClient, { IContributorAuditModel } from "../../api/SOOSHooksApiClient";
+import { SOOS_CONSTANTS } from "../../constants";
+import { ScmType } from "../../enums";
+import { soosLogger } from "../../logging";
+import GitHubAuditProvider from "./scm/GitHub/GitHubAudit";
+import FileSystem from "fs";
+import * as Path from "path";
+import { ParamUtilities } from "./utilities";
+
+export interface IContributorAuditProvider {
+  audit(implementationParams: Record<string, string | number>): Promise<IContributorAuditModel>;
+  validateParams(implementationParams: Record<string, string | number>): void;
+}
+
+class ContributingDeveloperAuditService {
```

Comment:
let's use ContributorAudit throughout
```suggestion
class ContributorAuditService {
```